README.md

```diff
@@ -391,21 +394,21 @@ The Miner is an Impostor who can shapeshift to warp to the last vent they were i
 Team : Impostors<br>
 Basis : Shapeshifter<br>
 
-The Grenadier is an Impostor who can shapeshift to blind nearby crewmates. During this time, you may kill undetected.<br>
+The Grenadier is an Impostor who can shapeshift to blind nearby crewmates. During this time, you can kill maybe undetected.<br>
```

Comment:
This sentence is a bit confusing. When I read this sentence this is how it sounds: As a Grenadier, I can shapreshift to blind nearby crewmates. During that time, there is a possibility I can kill without being detected.
If this is what you are trying to say, I would reword this to something like:
"The Grenadier is an Impostor who can shareshift to blind nearby crewmates. During this time, blinded crewmates will not detact your kills. Be careful, if other crewmates who may not be blinded when you kill."